src/Controls/src/Core/Handlers/Items/iOS/StructuredItemsViewController.cs

```diff
@@ -133,16 +133,18 @@ internal void UpdateSubview(object view, DataTemplate viewTemplate, nint viewTag
 			}
 
 			if (formsElement != null)
+			{
```

Comment:
These changes  on this file is just small cleanup and seems we need braces now :)